includes/orders/classes/class-refund-validator.php

```diff
@@ -264,7 +264,7 @@ public function validate_and_calculate_totals() {
 
 		// Overall refund total cannot be over total refundable amount.
 		$order_total = edd_get_order_total( $this->order->id );
-		if ( $this->total > $order_total ) {
+		if ( 1 === bccomp( $this->total, $order_total, 2 ) ) {
```

Comment:
@robincornett Does this need to use the order's currency decimal filter in order to work here? The last parameter is defined as:
`The optional scale parameter is used to set the number of digits after the decimal place which will be used in the comparison.`

If this is a cryptocurrency comparison, then the precision is 8, or even if there is a zero decimal currency, will this still hold true?

Reply:
Well, generally what EDD has at that point is something with two decimal places at the most.

However, it looks like `bccomp` isn't the right answer for us here--although it's not a new PHP function, all of the unit tests are failing, so I'm guessing that means that it's not necessarily available on every server. Thinking on this again.

Reply:
I don't think any of those failures are due to this code, looks like an HTTP request was failing from some of the test dependencies. I re-ran them and they are failing elsewhere now, that is code related.

EDD can't always get two decimals at that point if you are running a cryptocurrency, as they always have 8 decimal points, even if they are all zeros. This makes me think there is something up the chain that might not be multi-decimal currency compatible. Worth a look I think.

Reply:
Thanks--I do not have unit testing working on the new machine yet, so missed those. This gives me a direction.

Reply:
@robincornett I'm running Windows on my new machine and I have it rocking away, so maybe we can get on a call and figure it out on yours too.

I ended up finding a pretty good solution for local dev that didn't stray far from my Linux setup, actually.